packages/asc-ui/src/components/InfoBanner/InfoBanner.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import { Close } from '@datapunt/asc-assets'
+import InfoBannerStyle, {
+  Props,
+  CloseButton,
+  CloseButtonWrapper,
+  ContentWrapper,
+} from './InfoBannerStyle'
+import Heading from '../Heading'
+import CompactThemeProvider from '../../theme/CompactThemeProvider'
+import Paragraph from '../Paragraph'
+import Link from '../Link'
+import { ButtonVariants } from '../Button/ButtonStyle'
+
+const CLOSE_BUTTON_TITLE = 'Sluiten'
+
+const InfoBanner: React.FC<Props & React.HTMLAttributes<HTMLElement>> = ({
```

Comment:
Perhaps `BannerMessage` or `Alert` (like Bootstrap and ARIA) would be more more appropriate? Since the element can display warning and error messages as well.

Reply:
BannerMessage seems like a good one, Alert already is too descriptive for this component